source/sessions/tests/README.rst

```diff
@@ -78,7 +79,26 @@ the given session is *not* marked dirty::
         arguments:
           session: session0
 
+Snapshot session tests
+======================
+Snapshot sessions tests require server of version 5.0 or higher and 
+replica set or a sharded cluster deployment.
+Default snapshot history window on the server is 5 seconds. Running the test in debug mode, or in any other slow configuration
```

Comment:
The default was updated to 5 _minutes_ in SERVER-47855 so this section is probably not needed at all anymore.